app/models/outfit.rb

```diff
@@ -15,4 +15,11 @@ class Outfit < ApplicationRecord
 
   geocoded_by :address
   after_validation :geocode, if: :will_save_change_to_address?
+
+  include PgSearch::Model
+  pg_search_scope :search_by_title_occasion_and_description,
```

Comment:
Should line 20 be: "search_by_title_occasion_description_and_address" since address is included in your line 21